keydb.conf

```diff
@@ -1036,7 +1037,7 @@ lazyfree-lazy-user-del no
 #
 # io-threads 4
```

Comment:
We don't have io-threads so remove this section.  Instead add a comment indicating io-threads is an alias for server-threads